sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuRowToColumnarExec.scala

```diff
@@ -825,7 +825,7 @@ case class GpuRowToColumnarExec(child: SparkPlan, goal: CoalesceGoal)
     // The cudf kernel only supports up to 1.5 KB per row which means at most 184 double/long
     // values. Spark by default limits codegen to 100 fields "spark.sql.codegen.maxFields".
     // So, we are going to be cautious and start with that until we have tested it more.
-    if (output.length > 0 && output.length < 100 &&
+    if (Range(1, 100).contains(output.length) &&
```

Comment:
How is this faster?  Was this really a suggestion from the IDE?

Reply:
https://github.com/scala/scala/blob/8cc248dc1305df4c17bb6b5738b700b60c9b5437/src/library/scala/collection/immutable/Range.scala#L372-L387

Reply:
This stuck out to me too thanks for commenting

Reply:
IDE flagged it for a different thing, it likes replacing length > 0 with nonEmpty, which does not make sense here. I just rewrote to look more like Scala since I already had to look at it. it's not faster, it should be the same.